android/java/org/chromium/chrome/browser/notifications/BraveNotificationWarningDialog.java

```diff
@@ -117,11 +138,11 @@ private void init(View view) {
         clickOnNotNow(view);
     }
 
-    public boolean isBraveRewardsEnabled() {
-        return OnboardingPrefManager.getInstance().isBraveAdsEnabled();
+    public static boolean isBraveRewardsEnabled() {
```

Comment:
are we sure to check this function after native initialization ?

Reply:
This flow actually starts from BraveActivity::finishNativeInitialization
It's sure that we are calling after native initialization.